### DISCUSSION TITLE
Cant install

### DISCUSSION THREAD

**Discussion by lucky3582** [BurntSushi/ripgrep#1955] [Q&A] [answered]
Hi I tried to install ripgrep in centos but it failed
````
sudo yum install ripgrep
No package ripgrep available.
````
I am using centOS 
```
uname -r
3.10.0-1160.36.2.el7.x86_64
````

I tried this but this also failed
```
 yum-config-manager --add-repo=https://copr.fedorainfracloud.org/coprs/carlgeorge/ripgrep/repo/epel-7/carlgeorge-ripgrep-epel-7.repo
```
Can you help me how to install it. Thanks.

**Comment by BurntSushi** [comment, marked as answer]
I'm not sure, sorry. I don't use centOS. Hopefully someone else can help and update the README.

It might help if you in include the version of centOS that you're using.

You could also consider downloading the Linux binary from github.
